app/views/teachers/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>New Teacher</h1>
+
+<%= form_tag '/teachers', method: :post do %>
+
+<div>
+  <%= label_tag :name %>
+  <%= text_field_tag :name %>
+</div>
+
+<div>
+  <%= label_tag :bio %>
+  <%= text_field_tag :bio %>
+</div>
+
+<div>
+  <%= label_tag :certifications %>
+  <%= text_field_tag :certifications %>
+</div>
+
```

Comment:
This looks great. But you forgot a very **important** part of a form-- the submit button. Without it, a user doesn't know if their information was saved.